docs/src/development/troubleshoot.md

```diff
@@ -118,6 +118,17 @@ You can write to disk a `build` hook to generate anything you need later.
 Or you can declare writable [mounts](../configuration/app/storage.md#mounts), which are writable even during and after deploy.
 They can be used for your data: file uploads, logs, and temporary files.
 
+## Git push fails due to lack of disk space
+
+If you see the following message when attempting to do a `git push`:
+`There is not enough free space to complete the push`
+
+It usually indicates that large files are present in the repository (where they shouldn't be).
+Make sure that the paths for media files, dependencies, or databases, ... are set to be ignored in the `.gitignore` file.
```

Comment:
```suggestion
Make sure that the paths for files like media files, dependencies, and databases are set to be ignored in your `.gitignore` file.
```

The ellipses is essentially like a less clear version of `etc.`, which [Google recommends against](https://developers.google.com/style/word-list?hl=en#etc).